plugin/src/py/android_screenshot_tests/pull_screenshots.py

```diff
@@ -250,6 +250,9 @@ def pull_screenshots(process,
 
     temp_dir = temp_dir or tempfile.mkdtemp(prefix='screenshots')
 
+    if not os.path.exists(temp_dir):
+        os.makedirs(temp_dir)
```

Comment:
I don't think this script should be creating the directory.. if you're specifying an output directory, wouldn't it also be your responsibility to make sure the directory exists?